src/backend/gl/src/queue.rs

```diff
@@ -1077,6 +1077,18 @@ impl hal::queue::RawCommandQueue<Backend> for CommandQueue {
         Ok(None)
     }
 
+    #[cfg(any(feature = "glutin", feature = "wgl", target_arch = "wasm32"))]
+    unsafe fn present_surface(
+        &mut self,
+        surface: &mut crate::Surface,
+        _image: crate::SurfaceImage,
+    ) -> Result<Option<hal::window::Suboptimal>, hal::window::PresentError> {
+        #[cfg(all(feature = "glutin", not(target_arch = "wasm32")))]
+        surface.context.swap_buffers().unwrap();
```

Comment:
I don't think this work that easily. How would the backend know when it should render into the default framebuffer?

Reply:
you are right, GL backend needs more stuff...